packages/azure-http-specs/specs/azure/resource-manager/resources/extension.tsp

```diff
@@ -0,0 +1,554 @@
+import "@typespec/http";
+import "@typespec/rest";
+import "@azure-tools/typespec-azure-core";
+import "@azure-tools/typespec-azure-resource-manager";
+import "@typespec/spector";
+
+using TypeSpec.Http;
+using TypeSpec.Rest;
+using Azure.Core;
+using Azure.ResourceManager;
+using TypeSpec.OpenAPI;
+using Spector;
+
+namespace Azure.ResourceManager.Resources;
+
+model ExtensionsResource is ExtensionResource<ExtensionsResourceProperties> {
+  ...ResourceNameParameter<ExtensionsResource>;
+}
+
+/** ExtensionsResource properties */
+model ExtensionsResourceProperties {
+  @doc("The description of the resource.")
+  description?: string;
+
+  /** The status of the last operation. */
+  @visibility("read")
+  provisioningState?: ProvisioningState;
+}
+
+/** The interface of extensions resources,
+ * it contains 4 kinds of scopes (resource, resource group, subscription and tenant)
+ */
+@armResourceOperations
+interface ExtensionsResources {
+  @scenario
+  @scenarioDoc("""
+    Resource GET extension resource by tenant.
```

Comment:
Could you add one line above all these cases, something like

```
This test is passed by calling the API 4 times, by providing different parameters.
```

Reply:
Fixed by [0f68d4a](https://github.com/Azure/typespec-azure/pull/1924/commits/0f68d4a1e24215faa82efaa68f523c135a12963c)